docs/docs/source-plugin-tutorial.md

```diff
@@ -347,7 +347,7 @@ Experiment with different options in your `gatsby-config.js` file to see how tha
 
 Don't publish this particular plugin to npm or the Gatsby Plugin Library, because it's just a sample plugin for the tutorial. However, if you've built a local plugin for your project, and want to share it with others, `npm` allows you to publish your plugins. Check out the npm docs on [How to Publish & Update a Package](https://docs.npmjs.com/getting-started/publishing-npm-packages) for more info.
 
-> **NOTE:** Once you have published your plugin on `npm`, don't forget to edit your plugin's `package.json` file to include info about your plugin. If you'd like to publish a plugin to the [Gatsby Plugin Library](https://www.gatsbyjs.org/plugins/) (please do, it's really easy!), please follow the [Plugin Authoring guide](https://www.gatsbyjs.org/docs/plugin-authoring/#publishing-a-plugin-to-the-library).
+> **NOTE:** Once you have published your plugin on `npm`, don't forget to edit your plugin's `package.json` file to include info about your plugin. If you'd like to publish a plugin to the [Gatsby Plugin Library](https://www.gatsbyjs.org/plugins/) (please do, it's really easy!), please follow the [Publishing a plugin to the library](https://www.gatsbyjs.org/docs/submit-to-plugin-library/).
```

Comment:
This sentence doesn't make sense anymore, because of the link change. Maybe mirror the new paragraph from `plugin-authoring`:

"If you'd like to publish your plugin to the [Gatsby Plugin Library](/plugins/) (please do!), [follow these steps](/docs/submit-to-plugin-library/)."

Also I realize in the existing doc (already in the codebase) the internal links were accidentally not relative -- please make them relative!